webapp/src/rtcpeer/index.ts

```diff
@@ -33,14 +33,11 @@ export default class RTCPeer extends EventEmitter {
         }
     }
 
-    private onConnectionStateChange() {
-        switch (this.pc?.connectionState) {
+    private onICEConnectionStateChange() {
+        switch (this.pc?.iceConnectionState) {
         case 'connected':
             this.emit('connect');
             break;
-        case 'disconnected':
-            this.emit('disconnect');
-            break;
```

Comment:
Removing this for the time being since we are not making use of it. Should come back if/when we start to support ICE reconnects.